.pydocstylerc

```diff
@@ -1,5 +1,6 @@
 [pydocstyle]
 convention = pep257
+match = .*\.py
```

Comment:
basic question, why is this line necessary if you've changed the workflow yaml to `pydocstyle gnomad tests`?

Reply:
The default for `match` ignores files named `test_*.py`. So without this, `pydocstyle tests` doesn't actually check the test files contained in the `tests` directory.

http://www.pydocstyle.org/en/stable/usage.html#cli-usage